### PR TITLE
Enable ostree plugin in luna

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: playbooks/luna.yml [hunk 1]
```diff
@@ -21,6 +21,7 @@
       - "--enable-foreman-cli-remote-execution"
       - "--enable-foreman-cli-tasks"
       - "--enable-foreman-cli-templates"
+      - "--enable-ostree"
```
Comment (ekohl) [inline playbooks/luna.yml]: Does this work? AFAIK we don't have an ostree class. Don't you mean:
```suggestion
      - "--katello-ostree true"
```

File: playbooks/luna.yml [hunk 2]
```diff
@@ -21,6 +21,7 @@
...
     foreman_installer_additional_packages:
       - tfm-rubygem-foreman_virt_who_configure
```
Comment (ekohl) [inline playbooks/luna.yml]: https://github.com/theforeman/foreman-installer/pull/371 should expose these too

**Comment by ares** [comment]
`--katello-enable-ostree` is the right option I believe

File: playbooks/luna.yml (hunk 2 above)
Comment (ares) [reply playbooks/luna.yml]: I agree, can you add it there please? I think I could merge that PR in return :-)

**Comment by ares** [comment]
fyi @rplevka

**Pull request merged**